_guide/_pages/security.md

```diff
@@ -4,4 +4,79 @@ sidenav: security
 sticky_sidenav: true
 ---
 
-Security is everybody's responsibility at TTS. There are practices that we should adhere to as much as possible when building websites and this guide contains the ones that front-end designers and developers need to be aware of.
+Security is everybody's responsibility at TTS but if you're not used to thinking about security in your day-to-day job, 
+that's understandable. We come from a range of different backgrounds, some of which involved security more than others. 
+
+There are practices that we should adhere to as much as possible when building websites and this guide contains ones 
+that front-end designers and developers need to be aware of. We aim to give everyone a framework to think about security, and
+ways to approach it with our partners.
+
+This is an ever-expanding list; if an important issue is unrepresented, please feel free to open up a PR with your expertise 
+or add an issue!
+
+## FISMA
+Every system you'll work on at TTS has a FISMA level of impact. 
```

Comment:
[Before You Ship](https://before-you-ship.18f.gov/ato/) directs folks to [the OpenControl Introduction to ATOs](https://atos.open-control.org/categorization/). Could we merge this content there? (I'm a maintainer there as well.)

Reply:
Can you be a little more explicit about what "can we merge this content there" means? 

I'm happy to link there or move some content over there, but i'd like there to be enough context here that someone can get it without leaving the page.